wpaffiliatemanager/source/Pages/TemplateResponse.php

```diff
@@ -19,8 +19,16 @@ public function __construct($templateName, $viewData = array())
 
 	public function render()
 	{
+		$templates_path_default = WPAM_BASE_DIRECTORY . "/html/";
+		$templates_path = apply_filters('wpam_templates_path', $templates_path_default);
```

Comment:
This should normalize the path with trailingslashit()

```suggestion
		$templates_path = apply_filters('wpam_templates_path', $templates_path_default);

		// Normalize the path to ensure it ends with a trailing slash.
		$templates_path = trailingslashit($templates_path);
```